### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by DilsherB**
in this PR:
1. Home Page added.
2. NavBar added to navigate through the app.
3. styles added for all three pages i.e. Home Page, Calculator, Quote
4. Routes added and implemented.
5. Bootstrap used to style the components.
6. Made sure that there are no linter errors.
7. Pages are loading without refreshing the app.

### CODE REVIEW

**Review by joel-kalema — COMMENTED** [review]
Hi @DilsherB 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request  🛑 

pirates-of-the-caribbean-johnny-depp


- [ ] You have submitted a project that doesn't have a pull request description. Kindly add it 👍 


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @joel-kalema in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/components/Menu.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  return (
+    <div className="d-inline-flex px-3 w-100 bg-light">
+      <h5>Math Magicians</h5>
+      <ul className="d-flex justify-content-end gap-5 w-100 mt-2">
+        {Object.keys(menuItems).map((item) => (
+          <li key={item} className="list-group-item">
+            <a href={`${item}`} className="text-decoration-none text-dark">
+              {menuItems[item]}
+            </a>
+          </li>
```
Comment (DammyShittu) [inline src/components/Menu.js]: - [ ] Your app refreshes when I navigate across the pages. That should not be as this is a single-page app. Meaning we should be able to navigate the app without reloading it. You should consider using [NavLink from the `react-router-dom`](https://reactrouter.com/en/main/components/nav-link) package.

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @DilsherB,

## Highlights
✔️ Navbar feature

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Menu.js (hunk 1 above)
Comment (DilsherB) [reply src/components/Menu.js]: @DammyShittu Thank you for your guidance and kind review

**Review by BregornOriginal — APPROVED** [review]
Hi @DilsherB 

Your project is complete!  There is nothing else to say other than... it's time to merge it 

Congratulations! 

giphy (7)

## Highlights

✅ Linters are passing
✅ Good commit history
✅ Good PR title and description
✅ Excellent work using react-router 🥇 

## Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.
Cheers and Happy coding!:aplauso::aplauso::aplauso:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, remember to tag me in your question so I can receive the notification.
------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)[.](https://airtable.com/shrQAqnBwek5a0O0s)._)

**Pull request merged**